README.md

```diff
@@ -22,10 +22,16 @@ Follow the instructions to register a new GitHub app.
 
 The app is continuously deployed using [Vercel's GitHub app](https://github.com/apps/vercel).
 
-## How it works
+### How it works
```

Comment:
I'd suggest to leave the `h2` here, and move the new `### considerations` section below `## Deployment`, as both the points are related to Vercel deployment

Reply:
```suggestion
## How it works
```